addon/src/teardown-context.ts

```diff
@@ -1,5 +1,7 @@
 import type { TestContext } from './setup-context';
-import settled from './settled.ts';
+import Ember from 'ember';
```

Comment:
we cannot import from `ember` as it's planned to be deprecated soonish in the v6 series.

See: https://github.com/emberjs/rfcs/pull/1003

the RFC also has a big list of alternate ways to get what you need though.

Reply:
Good to know. This isn't new though, just moved from `setup-context.ts` and there are several other imports currently in the repo.

Also this project is setting `Ember.testing`, I only see an alternative for checking it.